python.py

```diff
@@ -0,0 +1,3 @@
+def square(x):
+""" This function will square any number"""
```

Comment:
this doc string is not very descriptive, could you please improve by .................